docs/root/intro/arch_overview/load_balancing.rst

```diff
@@ -20,7 +20,10 @@ Supported load balancers
 Round robin
 ^^^^^^^^^^^
 
-This is a simple policy in which each healthy upstream host is selected in round robin order.
+This is a simple policy in which each healthy upstream host is selected in round
+robin order. If weights are assigned to endpoints in a locality, then a weighted
+round robin schedule is used, where higher weighted endpoints will appear more
```

Comment:
The implementation is somewhat novel and not trivial. Should we describe here and potentially move some of the doc details I had asked for in the PR to here? I don't feel strongly either way.

Reply:
The effect though is standard weighted RR behavior, so I don't think we need to go into implementation detail in the user docs.

Reply:
OK fair enough.